### ISSUE TITLE
VSTest integration does not respect SkipAutoProps setting

### ISSUE THREAD

**Issue by StefanOssendorf** [coverlet-coverage/coverlet#959]
Hi,
the VSTest integration does not ignore auto properties during code coverage.
You can finde [here](https://github.com/StefanOssendorf/mwe-coverlet-skipAutoProps-not-working-as-expected) a MWE with a folder where I have generated a report from the coverage file (coverage.cobertura.xml in the linked rep).

According to this [VSTest integration doc](https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/VSTestIntegration.md#advanced-options-supported-via-runsettings) the skipping of auto properties should be supported.

As a preview from my test repo:
```csharp
public class Class1
{
    public String Name { get; set; }
    public Int32 Age { get; set; }

    public Boolean DoFunnyThings()
    {
        Name = "Dummy";
        Age = 1337;

        return true;
    }
}
```

Name and Age are counted for coverage which I think should not be the case.

If you need any more information just ask :-)

**Comment by Malivil** [comment]
This was asked for in #328 and merged in #912 on August 3rd
The last release was in May so this feature hasn't been included in a release yet

I'm looking forward to it too =)

**Comment by StefanOssendorf** [comment]
Well... I am embarassed now 😅.
Thanks for pointing that out!
